packages/webdriverio/src/commands/browser/getPuppeteer.ts

```diff
@@ -56,20 +56,33 @@ export default async function getPuppeteer (this: WebdriverIO.Browser) {
     if (cdpEndpoint) {
         this.puppeteer = await puppeteer.connect({
             browserWSEndpoint: cdpEndpoint,
-            // @ts-ignore ToDo(@L0tso): remove once https://github.com/puppeteer/puppeteer/pull/6942 is merged
+            // @ts-ignore ToDo: remove once https://github.com/puppeteer/puppeteer/pull/6942 is released
+            defaultViewport: null
+        }) as any as PuppeteerBrowser
+        return this.puppeteer
+    }
+    /**
+     * attach to a Selenoid\Moon CDP Session if there are Aerokube vendor capabilities
+     */
+    const requestedCapabilities = (this.requestedCapabilities as Capabilities.W3CCapabilities)?.alwaysMatch || this.requestedCapabilities as Capabilities.DesiredCapabilities
+    const isAerokubeSession = requestedCapabilities['selenoid:options'] || requestedCapabilities['moon:options']
+    if (isAerokubeSession) {
+        const { hostname, port, path } = this.options
+        this.puppeteer = await puppeteer.connect({
+            browserWSEndpoint: `ws://${hostname}:${port}${path}/devtools/${this.sessionId}`,
```

Comment:
Here is the problem. I checked such a case in my project, where we get the session along the path `/wd/hub/`. I am getting an error when I try to establish a WS connection. So it seems that browserWSEndpoint needs to be replaced with
```
`ws://${hostname}:${port}/devtools/${this.sessionId}`
```

Reply:
Good catch. I will check and fix it.